src/ui-services/notificationsService.js

```diff
@@ -23,6 +23,7 @@ angular.module('openshiftCommonUI').provider('NotificationsService', function()
       }
 
       notifications.push(notification);
+      $rootScope.$emit('showToastNotification', notification);
```

Comment:
instead of showToastNotification i think it would be cleaner to say newNotification or notificationAdded or something, in case something else was going to want trigger off of this later?